ga4gh/cli.py

```diff
@@ -255,15 +255,20 @@ def addGlobalOptions(parser):
     parser.add_argument(
         "--dont-use-reloader", default=False, action="store_true",
         help="Don't use the flask reloader")
+    parser.add_argument(
+        "--disable-urllib-warnings", default=False, action="store_true",
+        help="Disable urllib3 warnings")
 
 
 def server_main(parser=None):
     if parser is None:
         parser = argparse.ArgumentParser(
             description="GA4GH reference server")
-    addGlobalOptions(parser)
+    addServerOptions(parser)
     args = parser.parse_args()
-    frontend.configure(args.config_file, args.config, args.port)
+    frontend.configure(
```

Comment:
Why pass this in to the configure function? I think we should just call `` requests.packages.urllib3.disable_warnings()`` directly here rather than within the server configuration. This way, it's explicitly something we only do using the development CLI. If we make it an option for the configure function then people could abuse it in production. Disabling these warnings on a production deployment is a bad idea, and we shouldn't facilitate it.